.github/workflows/main.yml

```diff
@@ -19,10 +19,9 @@ jobs:
           - gcc
           - clang
         meson_options:
+          # Build options per compiler here
+          - ''
```

Comment:
i think we can just drop the `meson_options` altogether - whoever has to add them in the future can look up the git history or figure it out anew :)